lib/pager_duty/connection.rb

```diff
@@ -180,6 +207,8 @@ def initialize(token, token_type: :Token, url: API_PREFIX, debug: false)
         conn.use RaiseApiErrorOnNon200
         conn.use RaiseFileNotFoundOn404
         conn.use RaiseRateLimitOn429
+        conn.use RaiseForbiddenOn403
+        conn.use RaiseUnauthorizedOn401
```

Comment:
@technicalpickles I was wondering if it's easier to merge these error handlers into one handler, it'll make it easier to patch for consumers of the gem, and may clean up the stack a bit. 

If you're OK with that, I'll make a new PR after this one